converter/shieldhit/beam.py

```diff
@@ -1,19 +1,28 @@
 from dataclasses import dataclass
 import math as m
+from enum import Enum
+
+class DefinitionType(Enum):
```

Comment:
```suggestion
class BeamType(Enum):
```

refactor other parts of the code as well.
This enum is also imported in the parser, therefore the name should reflect to what kind of config it corresponds (beam/detect/geo)